src/OpenTelemetry.Instrumentation.Http/Implementation/HttpHandlerDiagnosticListener.cs

```diff
@@ -90,6 +90,7 @@ public override void OnStartActivity(Activity activity, object payload)
 
             activity.SetKind(ActivityKind.Client);
             activity.DisplayName = HttpTagHelper.GetOperationNameForHttpMethod(request.Method);
+            activity.SetCustomProperty("HttpHandler.Request", request);

```

Comment:
How does this relate to the enrichment stuff?

Reply:
In the demo I wanted to enrich by some context stuff and by adding userAgent from request & contentType from response. Surprised to notice that the raw objects were not available (they are for .NET Framework). So this is making the raw objects available to anyone downstream that might be interested.